### PR TITLE
style failed dat error #341

### PR DESCRIPTION
**Pull request by Kriesse**

### CODE REVIEW

File: elements/table-row.js [hunk 1]
```diff
@@ -323,20 +323,25 @@ function HexContent () {
...
-      <td class="tr cell-4 size">
-      </td>
-      <td class="cell-5">
+      <td class="cell-2" colspan="4">
+        <div class="cell-truncate color-red">
+          <h2 class="f6 normal">
+            Error
+          </h2>
+          <p class="f7">
+            Error initializing dat in ${err.dir}
```
Comment (karissa) [inline elements/table-row.js]: Can we say something less techy, like 'Could not share ${err.dir}'

Comment (Kriesse) [inline elements/table-row.js]: 👍

**Comment by juliangruber** [comment]
The build seems to fail because some of the copy was verified in the integration tests:

```js
 t.ok(val.indexOf('create new dat') > -1, 'has create new dat text')
```

You can fix those in `tests/welcome.js` and then verify locally by running `npm run test:integration`

**Comment by karissa** [comment]
ok i'll fix that

**Pull request merged**